cmd/antrea-agent/agent.go

```diff
@@ -479,8 +479,16 @@ func run(o *Options) error {
 	}
 
 	if features.DefaultFeatureGate.Enabled(features.Multicast) {
-		mcastController := multicast.NewMulticastController(ofClient, nodeConfig, ifaceStore)
-		if err := mcastController.Initialize(); err != nil {
+		multicastSocket, err := multicast.CreateMulticastSocket()
+		if err != nil {
+			return fmt.Errorf("failed to create multicast socket")
+		}
+		mcastController, err := multicast.NewMulticastController(ofClient, nodeConfig, ifaceStore, multicastSocket)
+
```

Comment:
The empty line should be before L486 to separate creating socket and controller, or no empty line at all. Not common to separate creating something and its error handling.

Reply:
updated. Removed empty lines.